### PR TITLE
Partition parser

### PR DESCRIPTION
**Pull request by rouga**
+ First Implementation of MPIHandler
+ Implementation of MeshPartition parser class with unit tests
+ Migration from Travis to Github action for CI to support MPI

### CODE REVIEW

File: include/parallelization/MPIHandler.hpp [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    /**
+     * @brief : Class to handle MPI communications between partitions
+     */
+    class MPIHandler {
+    public:
+
+        /**
+         *  @brief : ctor initializes MPI and populate rank,size
+         */
+        MPIHandler(int argc, char *argv[]);
```
Comment (abraham-bconstant) [inline include/parallelization/MPIHandler.hpp:19]: Maybe add the typical arguments in the description?

Comment (rouga) [reply include/parallelization/MPIHandler.hpp:21]: Done

**Comment by abraham-bconstant** [comment]
Adding a example on how to run the program in README.md would be great

File: include/parallelization/MPIHandler.hpp [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+    /**
+     * @brief : Class to handle MPI communications between partitions
+     */
+    class MPIHandler {
+    public:
+
+        /**
+         *  @brief : ctor initializes MPI and populate rank,size
+         *  parameters argc and argv are required by MPI_init function but they are
```
Comment (abraham-bconstant) [inline include/parallelization/MPIHandler.hpp:18]: should it be saying: ...but they are not used in this class ?

**Pull request merged**